lib/runnable.js

```diff
@@ -148,6 +148,14 @@ Runnable.prototype.isPending = function () {
 };
 
 /**
+ * Mark the runnable as pending or not pending
+ * @param pending
+ */
+Runnable.prototype.setPending = function (pending) {
```

Comment:
In most cases, it's better to use setters instead of using properties directly.

It's a common problem with using properties (and I encountered this problem while fixing this bug) that it's not easy to understand where someone changes the property.

You know that the property was changed, but you don't know when - all you can do is to debug the whole application. 
You can search for a string 'pending = ', but it can be a problem when several objects contain fields with the same name. 

if you have a setter, you can easily set a breakpoint inside the setter and find who called this method very quickly.